setup.cfg

```diff
@@ -29,11 +29,6 @@ project_urls =
 
 [options]
 packages = find:
-install_requires =
```

Comment:
@rxu17 one thing that's worth noting is that these are where you specify the dependencies of a package, so ideally these top level packages are specified and locked down instead of requirements.txt.

Then you don't necessarily need the `pip install -r requirements.txt`

Reply:
Note: the other ticket may have been misleading, but what I probably should've stated is lock these dependencies down

```
aacrgenie==16.3.0
pandas==...
synapseclient==...
```

Reply:
That said.. either way works. This is a Nit

Reply:
Hmmm yeah originally I went the `requirements.txt` route mainly because I was worried there may be other non-major dependencies that need to be locked down. Yeah it's easier to maintain if those are just the major dependencies, I'll try both ways and check.